packages/repo-tools/src/commands/repo/schema/openapi/lint.ts

```diff
@@ -47,6 +48,16 @@ async function lint(directoryPath: string, config?: { strict: boolean }) {
   const backstageRuleset = new Ruleset(
     {
       extends: [oas, ruleset],
+      rules: {
```

Comment:
is the linting necessary? If `allowReserved` is disabled by default maybe we should keep the same behaviour and just fix the plugins which are currently broken? 🤔

Reply:
What do you mean? I don't think the right fix here is double encoding client requests.

Reply:
usually certain characters are percent encoded to make sure that URLs are correctly interpreted. Otherwise things might go bananas when `/`, `#` or `?` are sent.
I'm worried that setting `allowReserved: true` everywhere might introduce other problems